src/DateTime/interfaces.py

```diff
@@ -99,7 +99,7 @@ def greaterThan(t):
         Python time module. Returns true if the object represents a
         date/time greater than the specified DateTime or time module
         style time.  Revised to give more correct results through
-        comparison of long integer milliseconds."""
+        comparison of integer milliseconds."""
```

Comment:
Here (and elsewhere in this file) it is actually **micro** (not *milli*) seconds

Reply:
ah yes, the same comment in DateTime.py was correct, it was updated when the code was updated millis -> micros, but not in the interface.

I fixed this in the same commit, I feel that's OK to do so because it's really minor.